core/plugins/pacemaker.py

```diff
@@ -0,0 +1,55 @@
+import re
+
+from core import checks
+from core.plugintools import PluginPartBase
+from core.cli_helpers import CLIHelper
+
+PACEMAKER_PKGS_CORE = ['pacemaker', 'crmsh', 'corosync']
+PACEMAKER_SVC_EXPR = ['pacemaker[a-zA-Z-]*',
+                      'corosync']
+
+
+class PacemakerBase(object):
+
+    @property
+    def crm_status(self):
+        return CLIHelper().pacemaker_crm_status
+
+    @property
+    def offline_nodes(self):
+        crm_status = self.crm_status()
+        for line in crm_status:
+            regex_match = re.search(r'.*OFFLINE.*\[\s(.*)\s\]',
+                                    line)
+            if regex_match:
+                return regex_match.group(1).split()
+        return None
```

Comment:
See my comment on your scenario but in short offline/online_nodes need to return [] if there aren't any so that operator.contains(nodes, <value>) does not fail.

Reply:
Fixed on c82b72d1a3be6ba00b6d2de421383f4c853ac395